src/main/resources/lib/ntb-import.ts

```diff
@@ -141,7 +139,7 @@ function pressReleaseToNtbArticle(pressRelease: PressRelease): NtbArticle {
     published: pressRelease.published,
     url: `https://kommunikasjon.ntb.no${pressRelease.url}`,
     publisherId: pressRelease.publisher.id,
-    channelId: pressRelease.channels[0].id,
+    channelId: pressRelease.channels.length > 0 ? pressRelease.channels[0].id : undefined,
```

Comment:
Kan det forekomme at `channels` er `undefined`, eller vil den alltid eksistere?

Reply:
Den kan være undefined, så ser jeg mangler et spørsmålstegn i sjekken 🙌

Reply:
Akkurat dette synes jeg er litt spennende, for jeg får feilmeldinger om jeg forsøker å gjøre `pressRelease.channels` til valgfri, men om jeg ikke gjør noe med det, så fungerer det fint selv om jeg ikke definerer noen channels 🤷

Reply:
Er ikke det litt mystisk da 😅 
Well, "if it works, don't fix it", da, eller? 😄

Reply:
Jo, veldig! Jeg lurer på det, også kan evt. Item kommentere/fikse det når jeg lager PR inn til dem? 😅 🤷